jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -395,27 +415,35 @@ YUI.add('juju-env-bakery', function(Y) {
       }
       // Check that the condition actually holds.
       check(plain.Condition);
-      var rootKey = sjcl.codec.base64.toBits(plain.RootKey);
-      
+      var rootKey = [];
```

Comment:
This seems wrong to me. newMacaroon says that its first argument must be an sjcl bitArray, but this is making a normal array. If this works, it's only coincidental and I don't think we should rely on that. As I suggested before, the right solution here is make the macaroon library use UInt8Arrays in its public interface.